pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -697,21 +697,16 @@ func describePodSummaryInline(rc *kapi.ReplicationController, includeEmpty bool)
 }
 
 func describePodSummary(rc *kapi.ReplicationController, includeEmpty bool) string {
-	actual, requested := rc.Status.Replicas, rc.Spec.Replicas
-	if actual == requested {
-		switch {
-		case actual == 0:
-			if !includeEmpty {
-				return ""
-			}
-			return "0 pods"
-		case actual > 1:
-			return fmt.Sprintf("%d pods", actual)
-		default:
-			return "1 pod"
+	switch rc.Status.Replicas {
+	case 0:
```

Comment:
I thought about it but I don't feel strong either way. Since we have no pods running currently it makes sense for me to show no output. Now `oc status` isn't just about the current state, and since we can provide info about future state derived from the spec, then we should do it. How about "scaling to # pods" when we start from zero?